rotary/rules/rotary.smk

```diff
@@ -221,17 +217,45 @@ rule build_gtdb_mash_ref_database:
         """
 
 
+rule set_up_sample_directories:
+    input:
+        "samples.tsv"
+    output:
+        long_reads = expand("{sample}/{sample}_long.fastq.gz", sample=SAMPLE_NAMES),
+        short_R1_reads = expand("{sample}/{sample}_R1.fastq.gz", sample=SAMPLE_NAMES),
+        short_R2_reads = expand("{sample}/{sample}_R2.fastq.gz", sample=SAMPLE_NAMES),
+    run:
+        def symlink_or_compress(in_file_path, out_file_path):
```

Comment:
I wonder if this function could be moved to utils -- what do you think?

Reply:
Done.